recipes/configs/llama2/1B_full_ppo.yaml

```diff
@@ -0,0 +1,182 @@
+# Config for single device RLHF full finetuning using PPO in ppo_full_finetune_single_device.py
+# using a TinyLlama2 1B model.
+#
+# This config uses hyperparameters based on small set of experiments and information
+# available from existing implementations.
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#   tune download TinyLlama/TinyLlama_v1.1 --hf-token <HF_TOKEN> --output-dir /tmp/TinyLlama_v1.1
+#   tune download smohammadi/tinyllama_rm_sentiment_1b  --hf-token <HF_TOKEN> --output-dir /tmp/tinyllama_rm_sentiment_1b --ignore-patterns ""
+#
+# You'll also need to ensure that {output_dir} exists beforehand, as checkpoints for policy and value models are saved in sub-folders.
+# To launch on a single device, run the following command from root:
+#   tune run ppo_full_finetune_single_device --config llama2/1B_full_ppo_single_device
+#
+# You can add specific overrides through the command line. For example
+# to override the checkpointer directory while launching training
+# you can run:
+#   tune run ppo_full_finetune_single_device --config llama2/1B_full_ppo checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
+#
+# This config works only for training on single device.
+
+# Tokenizer
+tokenizer:
+  _component_: torchtune.models.llama2.llama2_tokenizer
+  path:  /tmp/TinyLlama_v1.1/tokenizer.model
+
+# Dataset
+dataset:
+  _component_: torchtune.datasets.text_completion_dataset
+  source: trl-internal-testing/sentiment-trl-style
+  max_seq_len: null
+  split: train
+  column: prompt
+  add_eos: False
+
+
+# manually constructing 1B models
+policy_model:
+  _component_: torchtune.models.llama2.llama2
+  vocab_size: 32000
```

Comment:
Is it possible to just use the builder function for this? Or is there a reason why we need to build from here?

Reply:
Sorry, I'm missing something here - do we have a builder function for llama 1B? I opted for a longer yaml file vs. adding new model builders here.

Reply:
Removed 1B yaml. Going to add later in a deepdive/recipe cookbook.